docs/rst/manual/theory/ensemble_based_methods.rst

```diff
@@ -397,9 +397,97 @@ Iterative Ensemble Smoother - Ensemble subspace version
+
+is called tha Kalman gain, and :math:`M` is the linear measurement matrix, so that
+
+.. math::
+   \begin{align}
+   \hat{d} = M\mu_{\psi}
+   \end{align}
+
+is the best estimate of :math:`d` under the prior knowledge, and the error is assumed Gaussian with covariance :math:`\Sigma_d`. 
+The ensemble variants draw an :math:`N`-sample :math:`\{\psi\}_{i=1}^N` from the prior, 
+and perturbs observations :math:`d` using the distributions of measurements creating a corresponding observation-sample :math:`\{d\}_{i=1}^N`.
+The perturbations are guaranteed to sum to zero over the sample.
+A posterior sample is then formed from updating the prior with the equation for the posterior mean above
+
+.. math::
+   \begin{align}
+   \{\psi_i | d_i\} = \psi_i + \hat{K}(d_i-M \psi_i),
+   \end{align}
+
+where the estimated Kalman gain :math:`\hat{K}` is found by exchanging the prior covariance with an estimate based on its sample.
+Thus, the ensemble methods combine a sample from the prior with a sample from the likelihood of observed data, to form a new sample from the posterior.
+The posterior distribution that the posterior sample is conseptually sampled from, has mean and covariance found by
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{\psi|d} &= \bar{\psi} + \hat{K}(\bar{d}-M\bar{\psi}),\\
+   \hat{\Sigma}_{\psi|d} &= (I-\hat{K}M) \hat{\Sigma}_{\psi}
+   \end{align}
+
+From this, it is seen that when the sample size tends to infinity and estimates converge to the corresponding population quantities,
+then the ensemble variants converge to the standard Kalman filter in the linear Gaussian case.
+This convergence is however of a stochastic nature.
+
+More deterministic properties of the posterior are observed when the belief in measurements :math:`d` is varied. 
+Intuitively, when measurements have zero belief, i.e. unbounded variance, then the posterior should equal the prior.
+At the other end of the spectrum, if the measurements are perfect with zero variance,
+then the posterior estimate should equal the maximum-likelihood estimate, corresponding to a flat prior, 
+and as we are certain of the belief in this estimate (because the measurements are so amazing), 
+the determinant of the posterior covariance tends to zero from above.
+The maximum likelihood estimate is found by minimizing the relevant part of the negative log-likelihood of the data
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{ml} = \arg\min_{\mu} |d-M\mu|_2
+   \end{align}
+
+Furthermore, for a monotone sequence in belief in measurements, a monotone sequence
+for the posterior towards the likelihood estimate, and from the prior, should be observed.
+To summarize: 
+
+- For the posterior estimate, we require that
+  
+  a. The information in :math:`d` has been assimilated, creating a better estimate, so that :math:`|\hat{\mu}_{\psi|d}-\hat{\mu}_{ml}|_2<|\hat{\mu}_{\psi}-\hat{\mu}_{ml}|_2` and :math:`|\hat{\mu}_{\psi|d}-\hat{\psi}_{\psi}|_2<|\hat{\mu}_{\psi}-\hat{\mu}_{ml}|_2`.
```

Comment:
Does

:math:`|\hat{\mu}_{\psi|d}-\hat{\psi}_{\psi}|_2

represent the distance between the assimilated mean and the prior mean?
Not sure about the notation \hat{\psi}_{\psi}

Reply:
Thanks, this is a typo! It should be the prior sample expectation-estimate, so \hat{\mu}_{\psi}